### ISSUE TITLE
iocpreactor/udp.py:121 says 1234 when it could say ERROR_PORT_UNREACHABLE

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11037]
|[<img alt="ghazel's avatar" src="https://avatars.githubusercontent.com/u/31402?s=50" width="50" height="50">](https://github.com/ghazel)| @ghazel reported|
|-|-|
|Trac ID|trac#1500|
|Type|enhancement|
|Created|2006-02-21 11:59:02Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1500 1500
type__enhancement enhancement
reporter__ghazel ghazel
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__win32 win32
time__1140523142000000 1140523142000000
changetime__1143337309000000 1143337309000000
version__ 
owner__ 
cc__PenguinOfDoom cc__ghazel
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ghazel's avatar" src="https://avatars.githubusercontent.com/u/31402?s=50" width="50" height="50">](https://github.com/ghazel)|@ghazel commented|
|-|-|

```
#!html
<pre>
pretty trivial-
twisted 2.2.0
iocpreactor/udp.py line 121
says 1234 when it could just use the previously defined global 
ERROR_PORT_UNREACHABLE
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16455]) Merge iocp-multicast-1578

Author: ghazel
Reviewers: exarkun, itamarst
Fixes [#11108](https://github.com/twisted/twisted/issues/11108), [#11037](https://github.com/twisted/twisted/issues/11037)

Add multicast support to the IOCP reactor.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
